doc/install.rst

```diff
@@ -57,8 +57,8 @@ do), you will need at least one of the following:
 
 .. TODO: This can be broken down in a clearer way.
 
-* QtPy (PyQt4 or PyQt5)
-* PySide (Qt4/Qt5 alternative)
+* QtPy (PyQt5 or PyQt6)
+* PySide (pyside2 or pyside6)
```

Comment:
What's pyside2? That is not mentioned in the change log.

Reply:
It's their name for the Qt5 equivalent. I guess they jumped from pyside2 to pyside6 to follow Qt's versioning.